src/routes/safe/components/Apps/components/AddAppForm/index.tsx

```diff
@@ -66,6 +74,22 @@ const AddApp = ({ appList, closeModal }: AddAppProps): ReactElement => {
     <GnoForm decorators={[appUrlResolver]} initialValues={INITIAL_VALUES} onSubmit={handleSubmit} testId={FORM_ID}>
       {() => (
         <>
+          <AppDocsInfo>
+            <Text size="xl" as="span" color="secondary">
+              Safe Apps are third-party extensions.
+            </Text>
+            <Link
```

Comment:
Can we use ButtonLink component instead of Link + Icon?
Something like this:
`<ButtonLink textSize="xl" color="primary" iconType="externalLink" iconSize="sm">
        <Link
          size="xl"
          href="https://docs.gnosis.io/safe/docs/sdks_safe_apps/"
          target="_blank"
          rel="noopener noreferrer"
          title="More info about building Safe Apps"
        >
          Learn more about building Safe Apps
        </Link>
  </ButtonLink>`

Reply:
I think for external links we so far always use Link + icon, no? Buttons are used for internal links / actions where the user does not leave the interface.

Reply:
But agree with the linebreak